Microsoft.Toolkit.Uwp.UI.Controls/Menu/Menu.cs

```diff
@@ -57,6 +57,25 @@ public Orientation Orientation
                 new PropertyMetadata(Orientation.Horizontal, OrientationPropertyChanged));
 
         /// <summary>
+        /// Gets or sets the menu flyout placement
+        /// </summary>
+        public MenuFlyoutPlacement FlyoutPlacement
```

Comment:
I don't see why we need this new property and class? This should always be "auto". If the orientation is horizontal, then it's bottom or top (if the menu is at the bottom of the screen for some reason). If orientation is vertical it's either right or left